### PR TITLE
Update CLI endpoint alias

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | none |
| Versions      | 4.5, master |
| Edition       | n/a |

Update CLI configuration examples to account for a change in the alias.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale

### CODE REVIEW

File: docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md [hunk 1]
```diff
@@ -312,7 +312,8 @@ The new structure will then be:
          ez-all c55 ct1 l22 pl21 p1 p2 p21 p22
 ```
```
Comment (reithor) [inline docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md:313]:
```suggestion
```bash
```
bash formating is also used in other places.

**Review by reithor — CHANGES_REQUESTED** [review]

File: docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md [hunk 2]
```diff
@@ -291,14 +291,14 @@ In the event when a new version of `Child` is published, the following keys are
...
+Effectively, in this example HTTP cache for `[Parent1]` and `[Child]` are cleared.
 
 
 ### Tags purged on move event
 
 With the same Content structure as above, the `[Child]` location is moved below `[Parent2]`.
 
-The new structure will then be:
+The new structure is then:
 ```
```
Comment (reithor) [inline docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md:302]:
```suggestion
```bash
```

**Review by reithor — CHANGES_REQUESTED** [review]

**Review by reithor — APPROVED** [review]

File: docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md [hunk 3]
```diff
@@ -291,15 +291,16 @@ In the event when a new version of `Child` is published, the following keys are
...
 
 ### Tags purged on move event
 
 With the same Content structure as above, the `[Child]` location is moved below `[Parent2]`.
 
-The new structure will then be:
-```
+The new structure is then:
+
+```bash
```
Comment (adamwojs) [inline docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md:303]:
```suggestion
```
```

In this code block we don't have any particular language so I would leave it as plain text

Comment (dabrt) [reply docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md]: Vale reacts with a warning for posting code blocks without a highlighter. I am leaving it as you suggested.

**Comment by adamwojs** [comment]
@dabrt rebase is needed here

**Review by adamwojs — APPROVED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Review by julitafalcondusza — APPROVED** [review]

**Pull request closed without merging**